index.html

```diff
@@ -17,5 +17,61 @@
+            <figcaption>
+                Pay as you go
+            </figcaption>
+        
+        </figure>    
+            <p>
+            </p>
+            <section class="services">
+                <p>
```

Comment:
If these <p> tag aren't needed then they should be removed to make this a little cleaner!